docs/verifiable-document/wrapping-document.md

```diff
@@ -0,0 +1,97 @@
+---
+id: wrapping-document
+title: Wrapping Document
+sidebar_label: Wrapping Document
+---
+
+Every OA document has a checksum that provides it a tamper-proof property. At the same time, because the checksum can be used to uniquely identify a document, the checksum (or its derived value) is stored onto the document store as evidence of issuance. To compute the checksum, a `raw document` goes through a process known as `wrapping` to become a `wrapped document`. Only then, the document is ready to be issued onto the blockchain.
+
+Multiple documents can be wrapped at the same time in a single batch operation, creating a single checksum for the entire batch of raw documents. This is especially useful when using document store on the Ethereum blockchain to lower the transaction cost and time.
+
+In this guide, we will learn how to generate the checksum by running the `wrapping` process.
+
+## Pre-requisite
```

Comment:
I changed "Pre-requisite" to "Prerequisites" in the other docs!

Reply:
done